site/content/docs/03-run-time.md

```diff
@@ -280,34 +281,34 @@ Function that creates a store which has values that can be set from 'outside' co
 `update` is a method that takes one argument which is a callback. The callback takes the existing store value as its argument and returns the new value to be set to the store.
 
 ```js
-import { writable } from 'svelte/store';
+import { writable } from "svelte/store";
```

Comment:
we prefer single quotes everywhere in JS, so I don't think this should be changed. most of the other formatting changes look good, but would be much better as a separate PR. it's harder to review the actual change when I have to sift through dozens of lines of formatting